config/application.rb

```diff
@@ -116,6 +116,7 @@ def config.database_configuration
       plugin-third-party.js
       markdown-it-bundle.js
       service-worker.js
+      application-footer.js
```

Comment:
This is a terrible name (I tried 😢). This is supposed to be the home of all inline JS currently in `application.html.erb`. Suggestions more than welcome.

Reply:
preloaded-application-data.js ?

Reply:
Similar ones came across my mind too, only concern is that the file will contain some [non-data-related scripts](https://github.com/discourse/discourse/blob/879067d0009ad4db4669af59215a4921e75afbc1/app/views/common/_discourse_javascript.html.erb#L1-L35) too. I will rename to this for the time being tho.